budgetportal/settings.py

```diff
@@ -150,6 +151,15 @@
 AWS_S3_ENDPOINT_URL = os.environ.get("AWS_S3_ENDPOINT_URL", None)
 AWS_S3_REGION_NAME = os.environ.get("AWS_S3_REGION_NAME", None)
 
+if DEBUG:
+    AWS_DOMAIN = "localhost:9000"
```

Comment:
let's rather pass values in using per-variable environment variables. `if DEBUG` locks you in a bit and can have surprising effects

Reply:
Do you want to pass all of those values as env ariale?